### DISCUSSION TITLE
delete function from Ctrl + C menu

### DISCUSSION THREAD

**Discussion by qwertyaos** [psoukie/zipchord#138] [Ideas]
I would like to see a deleting function for a chord or a shorthand that may interfere with previous typing habits.
It can be achievable simply by not limiting blank input.
Thanks a lot for this program. Loving it.

**Comment by psoukie** [comment]
Makes total sense. I was thinking of eventually expanding the functionality of the Add Shortcut window to allow editing the chords, but didn't think of also having a way to remove them without editing the dictionary.

I created an issue to track progress on this request here: #140
